quickstart/app/Http/Controllers/Task_Controller.php

```diff
@@ -20,8 +20,7 @@ public function task(){
     public function post_Task(Request $rq){
       $rule=[
         'name'=>'required|max:255'];
-      $msg=['name.required'=>'Trường này không được bỏ trống'];
-      $val=Validator::make($rq->all(),$rule,$msg);
+      $val=Validator::make($rq->all(),$rule);
```

Comment:
convention này thanh niên: các toán tử cách nhau 1 space : $val = Validator...., sau các tham số là 1 space: make(param1, param2)